test/unit/SourceLocator/SourceStubber/PhpStormStubsSourceStubberTest.php

```diff
@@ -133,6 +133,16 @@ static function (string $className): bool {
                         return false;
                     }
 
+                    // Missing in JetBrains/phpstorm-stubs
```

Comment:
Can we reference an upstream issue, please?

Reply:
> Alternatively, I can do this later today, if you want

👍 Thank you.

Reply:
@kukulich I raised https://github.com/JetBrains/phpstorm-stubs/pull/1781 with the full list: can you update the various TODOs?

I don't care if the upstream PR ("issue") is closed, but at least I centralized this knowledge somewhere